[2024-11-08T18:39:03.188+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-08T18:39:03.207+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_detection.load_dependencies scheduled__2024-11-08T18:38:00+00:00 [queued]>
[2024-11-08T18:39:03.250+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_detection.load_dependencies scheduled__2024-11-08T18:38:00+00:00 [queued]>
[2024-11-08T18:39:03.250+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-11-08T18:39:03.273+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): load_dependencies> on 2024-11-08 18:38:00+00:00
[2024-11-08T18:39:03.292+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1816) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-08T18:39:03.294+0000] {standard_task_runner.py:72} INFO - Started process 1818 to run task
[2024-11-08T18:39:03.295+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fraud_detection', 'load_dependencies', 'scheduled__2024-11-08T18:38:00+00:00', '--job-id', '326', '--raw', '--subdir', 'DAGS_FOLDER/fraud_detection_dag.py', '--cfg-path', '/tmp/tmpjmnaehmp']
[2024-11-08T18:39:03.305+0000] {standard_task_runner.py:105} INFO - Job 326: Subtask load_dependencies
[2024-11-08T18:39:03.639+0000] {task_command.py:467} INFO - Running <TaskInstance: fraud_detection.load_dependencies scheduled__2024-11-08T18:38:00+00:00 [running]> on host 641085defe8e
[2024-11-08T18:39:03.940+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='fraud_team' AIRFLOW_CTX_DAG_ID='fraud_detection' AIRFLOW_CTX_TASK_ID='load_dependencies' AIRFLOW_CTX_EXECUTION_DATE='2024-11-08T18:38:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-08T18:38:00+00:00'
[2024-11-08T18:39:03.947+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-08T18:39:03.964+0000] {fraud_detection_dag.py:26} INFO - Chargement des dépendances depuis S3...
[2024-11-08T18:39:39.311+0000] {fraud_detection_dag.py:42} INFO - Modèle chargé avec succès
[2024-11-08T18:39:42.574+0000] {fraud_detection_dag.py:47} INFO - ETL chargé avec succès
[2024-11-08T18:39:42.781+0000] {python.py:240} INFO - Done. Returned value was: True
[2024-11-08T18:39:42.890+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-08T18:39:42.891+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=fraud_detection, task_id=load_dependencies, run_id=scheduled__2024-11-08T18:38:00+00:00, execution_date=20241108T183800, start_date=20241108T183903, end_date=20241108T183942
[2024-11-08T18:39:42.947+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-08T18:39:43.019+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-08T18:39:43.022+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
